index.js

```diff
@@ -58,6 +58,31 @@ function onInitHooks() {
 	)
 }
 
+function onInitCreateAdmin() {
+	const email = process.env.BALENA_ROOT_EMAIL || '';
+	const password = process.env.BALENA_ROOT_PASSWORD || '';
+
+	if (email == '' || password == '') {
+		return;
+	}
+
+	console.log("Creating admin account...")
+
+	const { runInTransaction } = require('./src/platform');
+	const { registerUser } = require('./src/platform/auth');
+
+	return runInTransaction(tx =>
+		registerUser({ username: 'root', email, password }, tx),
+	)
+	.then(() => {
+		console.log("Admin account created successfully!");
+	})
+	.catch(() => {
+		console.log("Admin account already exists!");
```

Comment:
I think that a specific error type will be thrown if it already exists (`ConflictError`?), so it'd probably be better to catch that one specifically

Reply:
OK cool - that's a good suggestion